### ISSUE TITLE
How to manually type in param for cformelements

### ISSUE THREAD

**Issue by GoogleCodeExporter** [loner02/vn-canvas#5]
```
TL;DR:
function drawskillbox () {
marquee(param)
}

How do I type in the param? Unlike in the script, I cannot do {name:"sample"} 
right?

Long version:
Hi, I want to create a function that will allow me to display certain cform 
elements only on a variable change, ie. displaying a level up screen only upon 
experience stat reaching experience required. The way I've thought to do this 
is to call a function that will draw the level up screen when Stats.lvl gets 
updated.

If I'm not doing it through the script but through a plugin, how do I do so?

Thanks!
```

Original issue reported on code.google.com by `tongrhj` on 19 Jul 2012 at 7:34
